### ISSUE TITLE
MacOS Launch Scripts

### ISSUE THREAD

**Issue by Aeonitis** [NationalSecurityAgency/ghidra#5040]
**Is your feature request related to a problem? Please describe.**
There are no launch scripts for MacOS devices [neither Apple Silicon nor Intel].

**Describe the solution you'd like**
A ghidraRun script on Ghidra/Runtimes/Mac.

**Describe alternatives you've considered**
I am sure I missed an alternative route, but I have made a draft script to help me fetch the dependencies, build a distribution locally, that they would not need to undergo any further time and effort in installing and running their own build.

**Additional context**
Prerequisite: The user has installed the required Gradle and Java versions on their Mac.

**Comment by ryanmkurtz** [comment]
The Linux scripts are designed to run on both Linux and macOS.

**Comment by Aeonitis** [comment]
Thanks for the quick response Ryan, but I just ran it and get this error.
"Ghidra cannot launch in development mode because Eclipse has not compiled its class files."

This is after I have built it. I am using IntelliJ, not Eclipse. Maybe I built the app the wrong way using the Linux script...

Feel free to criticize the need for an automated script to build locally and generate the run script [here](https://github.com/NationalSecurityAgency/ghidra/pull/5041).
